kiwi_post_run

```diff
@@ -91,13 +93,22 @@ echo "name $NAME"
 echo "version $VERSION"
 echo "release $RELEASE"
 echo "source $IMAGE"
+echo "metadata $METADATA"
+
+# Generate metada file for container-feeder
+cat << EOF > $METADATA
+Name: $CONTAINER_RAW
+Tag: $CONTAINER_TAG
+File: $IMAGE
+EOF
```

Comment:
I think it would be better to generate the `.metadata` file as a json one.

Moreover I think it's possible for an image to have multiple tags (eg: mariadb could have both tags `10` and `10.2.1`).